pkg/env/costmodelenv.go

```diff
@@ -42,6 +42,7 @@ const (
 
 	EmitPodAnnotationsMetricEnvVar       = "EMIT_POD_ANNOTATIONS_METRIC"
 	EmitNamespaceAnnotationsMetricEnvVar = "EMIT_NAMESPACE_ANNOTATIONS_METRIC"
+	EmitDeprecatedMetrics                = "EMIT_DEPRECATED_METRICS"
```

Comment:
Is it a good name? Any suggestions?

I assume there is more metrics that potentially can be deprecated in the future.
Reusing the same variable can be undesirable.

Reply:
I'd prefer something like "EmitAdditionalMetrics" or "EmitNonDefaultMetrics" to indicate they're not used by OpenCost directly but they're available. "Deprecated" implies they'd be removed later, I don't think that's the intention, just disabling them for most users.

Reply:
I'm considering eventually removing them from the code, viewing it as a temporary safety measure in case of issues.

Opencost currently provides a limited selection of kube-state-metrics. I believe we should focus on including only those metrics essential for Opencost. For a more comprehensive set of metrics, users should directly utilize kube-state-metrics.